### DISCUSSION TITLE
76X-78X Linking Fields Discussion

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#444] [General]
@pennylenger and @CECSpecialistI started mapping the 770 field and brought it up at the February 28th meeting, asking whether we should map to notes or mint entities for linking entry fields. The group said, "mint entities", and "we should talk about these more", and "we should have lots of examples"! Here is a place to discuss these fields in more depth. Hopefully with examples :) @AdamSchiff @lake44me @GordonDunsire @szapoun @corialanus @JianPLee @junghaelee @cspayne

**Comment by GordonDunsire** [comment]
Let's kick off with a response to the original questions:

For 770 - Supplement/Special Issue Entry (R) Full, these are notes on entities related to what is being described by the main MARC record.
Is it a good idea to map 770 in this way:
[Work] has note on work: “Supplement/Special Issue: [$a value]. Edition: [$b value] ($c value). Imprint: [$d value]. Related parts: [$g value] …etc.
Or is there any other suggested approach that I can take to deal with 770.

Response: These are more than just notes. They contain structured metadata about a relationship and related entity of the entity being described by the MARC record. It is therefore possible to transform the data into an RDA relationship element, a URI for the related entity, and a partial description sub-graph of the related entity; this is similar to [the approach I am taking with 6xx subject entry fields](https://docs.google.com/document/d/1T5VyAH6bPKBTJp_j4l2ecOLUlP00u-GK7Ubti58mtqU/edit).

So the basic model is:

<EntityDescribed> <RDA relationship> <instanceRelatedEntity> .
<instanceRelatedEntity> ...

770 example from MARC 21 manual:

245 	00$aJournal of cellular biochemistry.
770 	0#$tJournal of cellular biochemistry. Supplement$x0733-1959

=>

<Journal of cellular biochemistry [work]> rdawo:P10172 <Journal of cellular biochemistry. Supplement [work]> . // has supplement work
<Journal of cellular biochemistry. Supplement [work]> rdawd:P10366 "0733-1959" . // has ISSN

The "has supplement work" RDA relationship element is transformed from 770 ind2=#. If ind2!=#, subfield $i needs to be mapped to an RDA relationship element; this should be subsumed into the separate discussion on relationship elements.

[We cannot just mint the supplement work from the ISSN for reasons already discussed: not every ISSN has a corresponding URI, and then there are errors, etc.]

In general, the related entity data allows the determination of multiple related entities based on the aap; again, this is the same as for 6xx and other 7xx fields.

Note that I have not looked at relationships between the multiple related entities but, for example, I expect there to be a creator relationship between the person-entity part and the work-entity part of a personal name added entry. The same goes for 7xx; we need to develop a basic model.

**Comment by tmqdeborah** [comment]
@GordonDunsire says in his comment above "subfield $i needs to be mapped to an RDA relationship element; this should be subsumed into the separate discussion on relationship elements." 

I have begun looking into the WEMI-WEMI relationships. We have the RDA WEMI-WEMI relationship elements. Does anyone know whether there is a list of MARC labels and/or codes for these relationships?

**Comment by AdamSchiff** [comment]
There is this:

https://loc.gov/aba/rda/mgd/relationshipLabels/index.html
and
https://loc.gov/aba/rda/mgd/relationshipsSubject/index.html

The MARC relators and relationships are at https://id.loc.gov/vocabulary/relators.html
and
https://id.loc.gov/vocabulary/relationship.html

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by tmqdeborah** [comment]
Thanks @AdamSchiff

Oops, I should have said “We have mapping for the RDA WEMI-Agent relationships: from the MARC Relator codes and terms to RDA elements”. 

The Linking Fields and 6XX, 7XX and 8XX name/title fields are WEMI-WEMI. 

You have pointed me to the PCC Relationship Labels at https://loc.gov/aba/rda/mgd/relationshipLabels/index.html

As @GordonDunsire points out in his [comment](https://github.com/uwlib-cams/MARC2RDA/discussions/444#discussioncomment-8643725), they and any other WEMI-WEMI labels will need to be mapped.

I have put another spreadsheet <MrcRelationshipValuesMappedToRDA.xlsx> in the Headings Fields Folder at: https://drive.google.com/drive/folders/1PVipw3x09XQy7-ZLSfQCiOv2nL9TR1u1

This is just the start of something that should end up similar to the Relators spreadsheet <[MrcRelatorValuesMappedToRDA.20240229.xlsx](https://docs.google.com/spreadsheets/d/15CaDxSxdEhXkrHH3Aj_5g2S5shXJjtem/edit#gid=266732438)>.

It includes all RDA WEMI-WEMI relationships and their definitions, sorted by their Domains, plus a start at copying the PCC Relationship Labels that Adam pointed out and their definitions, with a ‘TRUE/FALSE’ comparison formula to indicate where the labels and definitions do not match. Note that we are definitely going to run into problems mapping these PCC labels because of their decision to 

We don’t seem to have a discussion page for the Headings Fields; so, I’m just letting you know that I have also put another spreadsheet < [HeadingsFieldsPersonalNames.20240305.xlsx](https://github.com/uwlib-cams/MARC2RDA/discussions/HeadingsFieldsPersonalNames.20240305.xlsx)> in the folder. Hopefully, this will help with our discussion, tomorrow, on these fields.
